src/includes/dockerfile/Pgpool-3.3-3.7.part.Dockerfile

```diff
@@ -23,15 +23,15 @@ RUN  wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O - | apt-key a
 RUN  apt-get install -y libffi-dev libssl-dev openssh-server
 
 {{ #PG_CLIENT_LATEST }}
-RUN  apt-get install -y postgresql-client-{{ PG_CLIENT_VERSION }}={{ PG_CLIENT_PACKAGE_VERSION }}\* 
+RUN  apt-get install -y postgresql-client-{{ PG_CLIENT_VERSION }}
 {{ /PG_CLIENT_LATEST }}{{ ^PG_CLIENT_LATEST }}
 RUN TEMP_DEB="$(mktemp)" && \
     wget -O "$TEMP_DEB"  "http://atalia.postgresql.org/morgue/p/postgresql-{{ PG_CLIENT_VERSION }}/postgresql-client-{{ PG_CLIENT_VERSION }}_{{ PG_CLIENT_PACKAGE_VERSION }}_amd64.deb" && \
     (dpkg -i "$TEMP_DEB" || apt-get install -y -f) && rm -f "$TEMP_DEB"
 {{ /PG_CLIENT_LATEST }}
 
 {{ #PGPOOL_LATEST }}
-RUN  apt-get install -y libpgpool0={{ PGPOOL_PACKAGE_VERSION }}\* pgpool2={{ PGPOOL_PACKAGE_VERSION }}\* 
+RUN  apt-get install -y libpgpool0 pgpool2
```

Comment:
Now we lost control over latest version of pgpool.
Assuming 3.8 is released, Dockerfile will install 3.8 while expected is 3.7
May be we can do `libpgpool0={{ PGPOOL_VERSION }}\* pgpool2={{ PGPOOL_VERSION }}\*`